RocketCast/EpisodeModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  EpisodeModel.swift
+//  RocketCast
+//
+//  Created by James Park on 2016-09-10.
+//  Copyright © 2016 UBCLaunchPad. All rights reserved.
+//
+
+import Foundation
+
+
+struct EpisodeModel {
+    
+
+    var title = NSMutableString()
```

Comment:
I still think that this should be `optional`. Swift has many ways of handling optionals and we should use it. Furthermore, there could be cases where the title is actually blank.